docs/source/en/perf_train_cpu.mdx

```diff
@@ -58,3 +58,103 @@ Take an example of the use cases on [Transformers question-answering](https://gi
 <b>--use_ipex \</b>
 <b>--bf16 --no_cuda</b></pre> 
 
+### Performance Examples
+
+[Bert-base-cased](https://huggingface.co/bert-base-cased?text=Paris+is+the+%5BMASK%5D+of+France.), [Distilbert-base-uncased](https://huggingface.co/distilbert-base-uncased?text=The+goal+of+life+is+%5BMASK%5D.) and [Bert-large-uncased](https://huggingface.co/bert-large-uncased?text=Paris+is+the+%5BMASK%5D+of+France.) are selected as examples with different model sizes to demonstrate the BFloat16 training performance achieved by IPEX and speed-up ratio against Stock PyTorch FP32 on the 3rd Generation Xeon® Scalable Processors (Intel® Xeon® Platinum 8380H CPU). All the benchmarkings are using [IPEX CPU launcher](https://intel.github.io/intel-extension-for-pytorch/cpu/latest/tutorials/performance_tuning/launch_script.html) to set system configs to ensure a best-to-best comparison, like numactl (--node_id 0 , i.e., one socket full cores) and jemalloc (--enable_jemalloc).
+
+
+The examples are refering to the [Text-classification](https://github.com/huggingface/transformers/tree/main/examples/pytorch/text-classification) (based on the script [run_glue.py](https://github.com/huggingface/transformers/blob/main/examples/pytorch/text-classification/run_glue.py)), and SST-2, STSB and RTE are scoped from the [GLUE](https://gluebenchmark.com/) benchmark of fine-tuning the library models for sequence classification (taking samples/second as the performance metric).
+
+#### Software configuration:
+| Software | Version|
+|-------|------------------------------|
+| PyTorch  | v1.12.1+cpu  |
+| IPEX  | v1.12.100+cpu  |
+|Transformers| [main](https://github.com/huggingface/transformers/commit/0ea53822f8bdc2c0c41b45bcd04fa6c031e5e700) |
+
+#### Training configuration:
+| GLUE Task  | Setups |
+|-------|------------------------------|
+| SST-2; STS-B; RTE |per_device_train_batch_size=128 ; max_seq_len=128 ; num_train_epochs=3 ; learning_rate=2e-5|
+
+#### Performance speed-up ratio:
+
+<table>
+<tbody>
+  <tr>
+    <th rowspan="2">Workload</th>
+    <th colspan="2">Training samples per second</th>
+  </tr>
+  <tr>
+    <th>Task</th>
+    <th>IPEX BFloat16  vs PT Float32</th>
+  </tr>
+  <tr>
+    <td style="text-align: center; vertical-align: middle" rowspan="10">Bert-base-cased</td>    
+  </tr>
+  <tr>
+    <td style="text-align: center; vertical-align: middle">SST-2</td>
+    <td style="text-align: center; vertical-align: middle">1.72x</td>
```

Comment:
This is still not a direct comparison of times. Could you display the times instead of a multiplier?

Reply:
Thanks for the suggestions, here have refined this section with several changes on the tables/model names.

For the comparison of times, the absolute time number may need an extra internal procedure if we really want to make them public,  and for now, we could only get the speed-up ratio to be public, here have refined the tables to a diagram. to better present the ratio.